contracts/routers/GlacisRouter.sol

```diff
@@ -10,7 +10,6 @@ import {AddressBytes32} from "../libraries/AddressBytes32.sol";
 
 error GlacisRouter__GMPNotSupported(); //0xed2e8008
```

Comment:
Could we remove this?

Reply:
It's required if one of our reserved GMPs isn't used yet

Reply:
I mean what's after "//0x", it breaks the error table generating script